### DISCUSSION TITLE
How to change version of fabric mods {like 20} from 1.19.2 to 1.20 ?

### DISCUSSION THREAD

**Discussion by BruhitsZain** [FabricMC/fabric-api#3412] [FAQ]
How to change version of fabric mods {like 20} from 1.19.2 to 1.20 ?
bcz  of error  INCOMPATIBLE MOD SET { Replace 'Minecraft' (minecraft) 1.20 with version 1.19.2.}
 yeah and if i change version of modsssss this problem should get fixed as i think......

If anyone got any solution plz tell. I'll appreciate .........

Incompatible mod set!
net.fabricmc.loader.impl.FormattedException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:

	 - Remove mod 'TLSkinCape' (tlskincape) 1.32 (C:\Users\ZAIN\AppData\Roaming\.minecraft\mods\tl_skin_cape_fabric_1.20_1.20.1-1.32.jar).
	 - Replace 'Minecraft' (minecraft) 1.20 with version 1.19.2.
	 - 
	 
Unmet dependency listing:

	 - Mod 'Fabric API' (fabric-api) 0.75.1+1.19.2 requires version 1.19.2 of 'Minecraft' (minecraft), but only the wrong version is present: 1.20!
	 - Mod 'Forge Config API Port' (forgeconfigapiport) 4.2.0 requires any version between 1.19.1 (inclusive) and 1.20 (exclusive) of 'Minecraft' (minecraft), but only the wrong version is present: 1.20!
	 - Mod 'Freecam' (freecam) 1.1.6 requires any 1.19.x version of 'Minecraft' (minecraft), but only the wrong version is present: 1.20!
	 - Mod 'Just Enough Items' (jei) 11.5.2.1007 requires version 1.19.2 of 'Minecraft' (minecraft), but only the wrong version is present: 1.20!
	 - Mod 'libIPN' (libipn) 2.0.1 requires any version between 1.19 (inclusive) and 1.19.3 (exclusive) of 'Minecraft' (minecraft), but only the wrong version is present: 1.20!
	 - Mod 'Litematica' (litematica) 0.12.6 requires any version between 1.19.1 (inclusive) and 1.20 (exclusive) of 'Minecraft' (minecraft), but only the wrong version is present: 1.20!
	 - Mod 'MaLiLib' (malilib) 0.13.0 requires any version between 1.19.1 (inclusive) and 1.20 (exclusive) of 'Minecraft' (minecraft), but only the wrong version is present: 1.20!
	 - Mod 'MiniHUD' (minihud) 0.23.3 requires any version between 1.19.1 (inclusive) and 1.20 (exclusive) of 'Minecraft' (minecraft), but only the wrong version is present: 1.20!
	 - Mod 'Pickable Villagers' (pickablevillagers) 1.3.3+1.19.2 requires any version between 1.19.2 (inclusive) and 1.20- (exclusive) of 'Minecraft' (minecraft), but only the wrong version is present: 1.20!
	 - Mod 'Puzzles Lib' (puzzleslib) 4.3.8 requires any version between 1.19.2 (inclusive) and 1.20 (exclusive) of 'Minecraft' (minecraft), but only the wrong version is present: 1.20!
	 - Mod 'Slot Cycler' (slotcycler) 4.0.2 requires any version between 1.19.2 (inclusive) and 1.20 (exclusive) of 'Minecraft' (minecraft), but only the wrong version is present: 1.20!
	 - Mod 'Fabric Command API (v2)' (fabric-command-api-v2) 2.2.1+413cbbc790 requires any version between 1.19-alpha.22.11.a (inclusive) and 1.20- (exclusive) of 'Minecraft' (minecraft), but only the wrong version is present: 1.20!
	 - Mod 'Wurst Client' (wurst) v7.35.2-MC1.19.2 requires any version between 1.19.2 (inclusive) and 1.20- (exclusive) of 'Minecraft' (minecraft), but only the wrong version is present: 1.20!
	at net.fabricmc.loader.impl.FormattedException.ofLocalized(FormattedException.java:51)
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:192)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:148)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

**Comment by TheBestAstroNOT** [comment]
You need to launch via minecraft fabric: 1.19.2, launcher version: 0.14.25/0.15
Else if u want to port to 1.20 u will have to download each and every manually again through curseforge/modrinth
